metsrw/mets.py

```diff
@@ -251,9 +251,11 @@ def _parse_tree_structmap(self, tree, parent_elem):
                 if file_elem is None:
                     raise exceptions.ParseError('%s exists in structMap but not fileSec' % file_id)
                 file_uuid = file_id.replace(utils.FILE_ID_PREFIX, '', 1)
-                group_uuid = file_elem.get('GROUPID').replace(utils.GROUP_ID_PREFIX, '', 1)
-                if group_uuid != file_uuid:
-                    derived_from = group_uuid  # Use group_uuid as placeholder
+                group_uuid = file_elem.get('GROUPID')
+                if group_uuid:
+                    group_uuid.replace(utils.GROUP_ID_PREFIX, '', 1)
```

Comment:
Oops, missed this in code review - `.replace` doesn't modify in place, but returns the updated value.  Fix would be
```python
group_uuid = file_elem.get('GROUPID', '').replace(utils.GROUP_ID_PREFIX, '', 1)
    if group_uuid != file_uuid:
    derived_from = group_uuid  # Use group_uuid as placeholder
```
or wait for the changes in dev/reingest to come in.